### ISSUE TITLE
DNS

### ISSUE THREAD

**Issue by bluesteal** [loki-project/loki-network-control-panel#83]
I have Lokinet Control Panel installed,

ran into some roadblocks compiling but was able to resolve by installing 

package "pkg-config" and compiling using the "-DBUILD_STATIC_DEPS=ON"

flag. My issue now is that .loki domains resolve just fine, but clearnet domains 

are not resolving via the exit node and the ip address does not reflect the exit 

node. Everything appears to be setup correctly, the exit node variable holds

 "exit.loki" and the exit node switch os turned on. Just tried this using stock 

Firefox. I have applied the known DNS fixes for Lokinet (both of them) to but 

the result was the same.

**Comment by bluesteal** [comment]
This is on Debain-10 btw :)

**Comment by majestrate** [comment]
is firefox using DoH?

**Comment by majestrate** [comment]
nvm .loki works so it's not using DoH

**Comment by majestrate** [comment]
i'll spin up a debian 10 vm then...

**Comment by bluesteal** [comment]
thank you sir

**Comment by majestrate** [comment]
i cannot reproduce, what does your `/etc/resolv.conf` contain?

**Comment by bluesteal** [comment]
nameserver 127.3.2.1

**Comment by majestrate** [comment]
actually, what does your `/var/lib/lokinet/lokinet.ini` contain that is probably more important.

**Comment by bluesteal** [comment]
I don't have a /var/lib/lokinet/ directory, let alone the lokinet.ini file

**Comment by majestrate** [comment]
how is lokinet running?

**Comment by bluesteal** [comment]
Yeah, not sure but I have two processes running lokinet and lokinet-gui. I think I started lokinet using systemd

**Comment by bluesteal** [comment]
I just have /usr/bin/lokinet as a directory currently

**Comment by bluesteal** [comment]
https://docs.loki.network/Lokinet/Guides/lokinet-linux-guide/ i followed this to install Lokinet and for the lokinet gui I followed the Github Readme

**Comment by bluesteal** [comment]
I reinstalled lokinet from github by building from source and have located the lokinet.ini file; the bind address is 127.3.2.1:53
